frontend/model/contracts/identity.js

```diff
@@ -18,10 +22,11 @@ import {
 } from './shared/validators.js'
 
 const attributesType = objectMaybeOf({
-  username: string,
+  username: stringMax(IDENTITY_USERNAME_MAX_CHARS, 'username'),
   email: string,
+  bio: optional(stringMax(IDENTITY_BIO_MAX_CHARS, 'bio')),
   picture: unionOf(string, objectOf({
-    manifestCid: string,
+    manifestCid: stringMax(MAX_HASH_LEN, 'manifestCid'),
```

Comment:
Just above this is a `string` that I assume is a URL. Feel free to set that to `2048` characters.